app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AdminController < ApplicationController
```

Comment:
Another option here for dealing with an Admin controller:

Often, our admin controllers deal with resources in the same way our regular controllers do - they handle shelters, and applications, and other records in our db. So, in order to make an application more extendable, it's usually easier to not having one single AdminController, but to have an admin *directory* and put new controllers inside of it depending on what functionality the admin needs to do. For example, for the admin shelter functionality, you can create a SheltersController inside the `admin` directory and it won't collide with the SheltersController that already exists outside the `admin` directory. You'll see more on this in your Advanced Routing class!

Reply:
This also allows better SRP adherence. Note how in this controller, you're dealing with lots of different things -- shelters, applications, etc. It's best for controllers to handle just one specific resource